index.html

```diff
@@ -96,12 +99,10 @@
         </label>
       </div>
       <label for="isNeedingBobtailInsurance">I need Bobtail Insurance.</label>
-      <div class="form-group antijump-wrapper" id="equipmentValueGroup">
-        <div class="form-label-group">
-          <input type="number" pattern="[0-9]*" inputmode="numeric" id="equipmentValue" name="equipmentValue" class="form-control" placeholder="$ Value of Truck">
-          <label for="equipmentValue">$ Value of Truck</label>
+      <div class="form-group" id="equipmentValueGroup">
```

Comment:
We lost the animated slide in/out transition related to `.antijump-wrapper` for the equipmentValue only when bobtail insurance is yes (as well as for the whole form slide-in).